app/assets/javascripts/mumuki_laboratory/application/avatar.js

```diff
@@ -0,0 +1,28 @@
+mumuki.load(function() {
+  let avatar_id = "";
+
+  $("#edit-avatar-icon").on('click', function(){
+    $('#avatar-picker').modal();
+  });
+
+  $("#user-form").on('submit', function(){
+    if (!avatar_id) {
+      const image_url = $('#user-avatar').attr('src');
+      $(this).append('<input type="hidden" name="user[image_url]" value="' + image_url + '"/>');
```

Comment:
We are already using `javascrit ${interpolations}` in other places of the code, so I'd suggest using them

Reply:
Correct